src/System.Diagnostics.Contracts/src/System.Diagnostics.Contracts.builds

```diff
@@ -4,9 +4,6 @@
   <ItemGroup>
     <Project Include="System.Diagnostics.Contracts.csproj" />
     <Project Include="System.Diagnostics.Contracts.csproj">
-      <TargetGroup>net46</TargetGroup>
```

Comment:
Actually I'm almost of the mind to delete all those comments since @mellinoe's refactoring is likely going to consolidate most of the projects and make the comments stale.  What do you think @mellinoe?